File: app/src/main/java/com/cheatdatabase/listeners/MyUnpublishedCheatsListItemSelectedListener.java
package com.cheatdatabase.listeners;

import com.cheatdatabase.data.model.UnpublishedCheat;

public interface MyUnpublishedCheatsListItemSelectedListener {
    void onRejectReasonButtonClicked(UnpublishedCheat cheat);

    void onEditCheatButtonClicked(UnpublishedCheat cheat);

    void onDeleteButtonClicked(UnpublishedCheat cheat);
}

File: app/src/main/java/com/cheatdatabase/listeners/MyUnpublishedCheatsListItemSelectedListener.kt
package com.cheatdatabase.listeners

import com.cheatdatabase.data.model.UnpublishedCheat

interface MyUnpublishedCheatsListItemSelectedListener {
    fun onRejectReasonButtonClicked(cheat: UnpublishedCheat)
    fun onEditCheatButtonClicked(cheat: UnpublishedCheat)
    fun onDeleteButtonClicked(cheat: UnpublishedCheat, position: Int)
}